### PR TITLE
Improve behave documentation

### PR DESCRIPTION
**Pull request by fgalan**
Including a quick-guide to set up the thing.

### CODE REVIEW

File: test/acceptance/behave/README.md [hunk 1]
```diff
@@ -1,44 +1,64 @@
...
+    to [SCL](https://www.softwarecollections.org).
+  * Python pip 1.4.1 or higher
+  * virtualenv 1.10.1 or higher
+  * Requeriments for the fabric Python module (installed in a next step), typically installed using: `yum install gcc python-devel`. [Fabric](http://www.fabfile.org) is a library and command-line tool for streamlining the use of SSH for application deployment or systems administration tasks.
+* Creating a virtual env (e.g. named orion_bh). Actually this step is optional (if you know what a virtual env is, then you would know which 
+  steps to skip in order to avoid using it ;)
 
-Note: We recommend the use of virtualenv, because is an isolated working copy of Python which allows you to work on a specific project without worry of affecting other projects.
+```
+virtualenv orion_bh
```
Comment (iariasleon) [inline test/acceptance/behave/README.md]: append how activate the virtualenv, 
```
source orion_bh/bin/activate
```

File: test/acceptance/behave/README.md [hunk 2]
```diff
@@ -1,44 +1,64 @@
...
+pip install -r requirements.txt --allow-all-external
 ```
 
+* Set properties.json
+
+```
+cp properties.json.base properties.json
+vi properties.json
+# Ensure that CB_LOG_FILE and CB_PID_FILE are ok in your environment
+# Ensure that CB_EXTRA_OPS
```
Comment (iariasleon) [inline test/acceptance/behave/README.md]: if the host is remote, ensure that CB_FABRIC_USER, CB_FABRIC_PASS or CB_FABRIC_CERT and CB_FABRIC_SUDO

File: test/acceptance/behave/README.md (hunk 1 above)
Comment (fgalan) [inline test/acceptance/behave/README.md]: It is already there. Look to line 25.

File: test/acceptance/behave/README.md (hunk 2 above)
Comment (fgalan) [inline test/acceptance/behave/README.md]: In the "quick way" it is assume that your are using your localhost (the idea is to be as simplest as possible). I will mentionan that at the beginning of the section.

Comment (fgalan) [inline test/acceptance/behave/README.md]: Fixed in cb56203bc3703aa57466d25d787be0c03ab3c86a

**Comment by iariasleon** [comment]
LGTM

**Pull request merged**